code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -465,6 +465,172 @@
 	icon_state = "grenade_phos_clf"
 	item_state = "grenade_phos_clf"
 
+/obj/item/explosive/grenade/sebb
+	name = "\improper G2 Electroshock grenade"
+	desc = "This is a G2 Electroshock Grenade. Produced by Armat Battlefield Systems, it's sometimes referred to as the Sonic Electric Ball Breaker, \
+		after a rash of incidents where the intense 1.2 gV sonic payload caused... rupturing. \
+		A bounding landmine mode is available for this weapon which activates a small drill to self-bury itself when planted. Simply plant it at your feet and walk away."
+	icon_state = "grenade_sebb"
+	item_state = "grenade_sebb"
+	/// Maximum range of effect
+	var/range = 5
+	/// Maximum possible damage before falloff.
+	var/damage = 110
+	/// Factor to mutiply the effect range has on damage.
+	var/falloff_dam_reduction_mult = 20
+	/// Post falloff calc damage is divided by this to get xeno slowdown
+	var/xeno_slowdown_numerator = 12
+	/// Post falloff calc damage is multipled by this to get human stamina damage
+	var/human_stam_dam_factor = 0.9
+
+/obj/item/explosive/grenade/sebb/get_examine_text(mob/user)
+	. = ..()
+	. += SPAN_NOTICE("To put into mine mode, plant at feet.")
+
+/obj/item/explosive/grenade/sebb/afterattack(atom/target, mob/user, proximity)
+	var/turf/user_turf = get_turf(user)
+	if(active)
+		return
+
+	if(!isturf(target))
+		return
+
+	if(user.action_busy)
+		return
+
+	if(target != get_turf(user))
+		return
+
+	if(locate(/obj/item/explosive/mine) in get_turf(src))
+		to_chat(user, SPAN_WARNING("There already is a mine at this position!"))
+		return
+
+	if(user_turf && (user_turf.density || locate(/obj/structure/fence) in user_turf))
```

Comment:
If a synth can't prime it, I don't think they should be able to plant it.

Replicate `Your programming prevents you from using this!` here.

Reply:
fixed